Formula/c/cucumber-cpp.rb

```diff
@@ -25,6 +25,12 @@ class CucumberCpp < Formula
   depends_on "asio"
   depends_on "tclap"
 
+  # Backport support for Asio 1.33+
+  patch do
+    url "https://github.com/cucumber/cucumber-cpp/commit/da6345bd1d0b0ac4cf4bc71866c98d55c72522a8.patch?full_index=1"
+    sha256 "1ad7f6513ee2d41c5625933b05249704c14484a7029acd330c5f665a11e6a66f"
+  end
```

Comment:
need new bottle for this commit?

Reply:
Maybe as one of modified headers is installed. Will also switch static libs to shared libs.